lib/date_time/date_time_core.c

```diff
@@ -174,6 +176,23 @@ int date_time_core_now(int64_t *unix_time_ms)
 	return 0;
 }
 
+int date_time_core_now_local(int64_t *local_time_ms)
+{
+	int err;
+
+	if (date_time_tz == DATE_TIME_TZ_INVALID) {
+		return -EAGAIN;
+	}
+
+	err = date_time_core_now(local_time_ms);
+	if (err) {
+		return err;
+	}
+
+	*local_time_ms += date_time_tz * 15 * 60 * 1000;
```

Comment:
```suggestion
	*local_time_ms += date_time_tz * 15 * MIN_PER_HOUR * MSEC_PER_SEC;
```
You can add the use of those constants to line 174 if you want but not necessary as it's old code.

Reply:
It should be SEC_PER_MIN, but I'm not going to add it since the version of the sdk that I'm on doesn't have those defines.

Reply:
> It should be SEC_PER_MIN

Of course 😄 

> but I'm not going to add it since the version of the sdk that I'm on doesn't have those defines.

Ok, I'll update those at some point.